x-pack/plugins/cloud_security_posture/common/constants.ts

```diff
@@ -36,9 +36,10 @@ export const VULNERABILITIES_INDEX_DEFAULT_NS =
 export const LATEST_VULNERABILITIES_INDEX_TEMPLATE_NAME =
   'logs-cloud_security_posture.vulnerabilities_latest';
 export const LATEST_VULNERABILITIES_INDEX_PATTERN =
-  'logs-cloud_security_posture.vulnerabilities_latest-*';
+  'logs-cloud_security_posture.vulnerabilities_latest*';
```

Comment:
I introduced a new mechanism for fetching data view fields, so I changed the pattern to include the `logs-cloud_security_posture.vulnerabilities_latest` index, otherwise, it would only fetch the pattern `logs-cloud_security_posture.vulnerabilities_latest-something`